app-backend/database/src/main/scala/com/azavea/rf/database/tables/ScenesToProjects.scala

```diff
@@ -111,8 +111,31 @@ object ScenesToProjects extends TableQuery(tag => new ScenesToProjects(tag)) wit
       )
     }
   }
+  
+  def setColorCorrectParamsBatch(projectId: UUID, params: BatchParams)(implicit database: DB) = {
+    val results = for {
```

Comment:
We should probably use a pinned session `withPinnedSession` instead of `.transactionally`, since that would allow you to check the number of rows updated in the middle of a transaction